compute/zonal_instance_group_manager/main.tf

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2022 Google LLC
+ * Copyright 2023 Google LLC
```

Comment:
We don't typically need to update the license year unless it's a correction.

Reply:
it is a correction, there was even a comment in my previous PR, but it got submitted anyway. 
https://github.com/terraform-google-modules/terraform-docs-samples/pull/485